pynuodb/encodedsession.py

```diff
@@ -393,9 +394,14 @@ def putScaledInt(self, value):
         @type value decimal.Decimal
         """
         #Convert the decimal's notation into decimal
-        value += 0
+        value += REMOVE_FORMAT
         scale = abs(value.as_tuple()[2])
         valueStr = toSignedByteString(int(value * decimal.Decimal(10**scale)))
+
+        #If our length is more than 9 bytes we will need to send the data using ScaledCount2
+        if len(valueStr) > 9:
```

Comment:
Sure thing, it would seem the issue is the way python handles sockets as byte strings are the only thing a socket can send. Before they used
```Python
     chr(protocol.SCALEDLEN0 + len(valueStr))
``` 
This starts us off with the base value of ScaledLen0 and you will add 1 for each character present in the string. The issue is they never checked overflow with ScaledLen8. So if you put something 10 numbers long or longer, it will overflow into the next protocol. I'm not sure if this is the best solution for sending numeric values in the first place